src/less/components/form.less

```diff
@@ -1,3 +1,39 @@
+@import "../global.less";
+@import "../core/normalize.less";
```

Comment:
Why do we import normalize into some component, and not others?

Reply:
Not all components require normalize.less